sdk/frontdoor/Azure.ResourceManager.FrontDoor/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-04-23)
```

Comment:
revert this change, this PR is for CDN, you should update this in the PR for front door